doc/TUTORIAL_SCRIPT.md

```diff
@@ -0,0 +1,92 @@
+# Script
+
+```shell script
+docker run -it elektra/elektra
+
+# Let's map the content of a hello.js file to Elektra's configuration in /hello
+sudo kdb mount hello.js /hello json
+
+# Set a value and retrieve it
+kdb set user/hello/hello/world "Hello World"
+kdb get user/hello/hello/world
+
+# You can also see where the file is located from Elektra
+kdb file user/hello/hello/world
+
+# You can edit the file and see what Elektra wrote
+vi /home/elektra/.config/hello.js
+>> Edit file to say "Hello World from File!"
+
+# Getting the value again ...
+kdb get user/hello/hello/world
+
+# You can also use Elektra's editor which additionally validates the file content
+kdb editor user/hello ini
+>> Edit file to say "Hello World from INI via kdb editor!"
+kdb get user/hello/hello/world
+
+# Elektra can export to various file formats
+kdb export user/hello
+kdb export user/hello yaml
+
+# You can search for plugins too
+kdb plugin-list
+kdb export user/hello tcl
+
+# Let's clean up our environment before we proceed
+kdb rm user/hello/hello/world
+
+# Removing the only entry also removed the whole file
+cat /home/elektra/.config/hello.ini
+
+# Remove the mountpoint too
+sudo kdb umount /hello
+
+# Let's see how configuration validation works
+# First let's write a specification file
+vi specification.ini
+>> []
+>> mountpoint = specification.ini
+>> [timeout]
+>> check/range = 0-100
+
+# Let's integrate the specification into the backend
+# The 'ni' plugin can be used to parse INI files into Elektra
```

Comment:
```suggestion
# The 'ni' plugin can be used to parse INI files for specifications
```

Make some meta-get meta-ls so that people see the difference.